### DISCUSSION TITLE
How much time to wait to access GitHub Copilot?

### DISCUSSION THREAD

**Discussion by NeXuS4Developer** [community/community#9218] [Copilot Conversations] [answered]
# is everyone can access GitHub Copilot?

Hey, i waiting for 1 week and i still doesn't have access to GitHub Copilot.
Please tell me if it's worth to wait or not everyone can access to it??

**Comment by DaniGalicia** [comment]
My request took around 1 month. Be patient

**Reply by NeXuS4Developer** [reply]
i am excited to get access to copilot,

**Comment by davecheney** [comment, marked as answer]
Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
